Sources/XCTest/XCTestCase.swift

```diff
@@ -308,4 +308,62 @@ extension XCTestCase {
             completionHandler(error)
         }
     }
+    
+    /// Creates and returns an expectation for a notification.
+    ///
+    /// - Parameter notificationName: The name of the notification the
+    ///   expectation observes for.
```

Comment:
nit-pick/typo: I think the "for" here is redundant, and could be removed. It should be "The name of the notification the expectation observes."